utilities/rpp-unittests/OCL/BatchPD_ocl.cpp

```diff
@@ -0,0 +1,729 @@
+        rppi_vignette_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, stdDev, noOfImages, handle);
+        break;
+    case 15:
+        test_case_name = "flip";
+        rppi_flip_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, flipAxis, noOfImages, handle);
+        break;
+    case 16:
+        test_case_name = "rain";
+        rppi_rain_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, rainPercentage, rainWidth, rainHeight, transparency, noOfImages, handle);
+        break;
+    case 17:
+        test_case_name = "rotate";
+        rppi_rotate_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxSize, angle, noOfImages, handle);
+        break;
+    case 18:
+        test_case_name = "warp-affine";
+        rppi_warp_affine_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxSize, affine_array, noOfImages, handle);
+        break;
+
+    case 19:
+        test_case_name = "resize";
+        rppi_resize_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxDstSize, noOfImages, handle);
+        break;
+
+    case 20:
+        test_case_name = "resize-crop";
+        rppi_resize_crop_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxDstSize, x1, x2, y1, y2, noOfImages, handle);
+        break;
+    case 21:
+        test_case_name = "Hue modification";
+        rppi_hueRGB_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, hueShift, noOfImages, handle);
+        break;
+    case 22:
+        test_case_name = "Saturation";
+        rppi_saturationRGB_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, saturationFactor, noOfImages, handle);
+        break;
+    case 23:
+        test_case_name = "Histogram Balance";
+        rppi_histogram_balance_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, noOfImages, handle);
+        break;
+    case 24:
+        test_case_name = "RandomShadow";
+        rppi_random_shadow_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, x1, y1, x2, y2, numbeoOfShadows, maxSizeX, maxSizey, noOfImages, handle);
+        break;
+    case 25:
+        test_case_name = "RandomCropLetterBox";
+        rppi_random_crop_letterbox_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxDstSize, x1, x2, y1, y2, noOfImages, handle);
+
+        break;
+    case 26:
+        test_case_name = "Absolute Difference";
+        rppi_absolute_difference_u8_pkd3_batchPD_gpu(d_input, d_input_second, srcSize, maxSize, d_output, noOfImages, handle);
+        break;
+    case 27:
+        test_case_name = "Accumulate";
+        rppi_accumulate_u8_pkd3_batchPD_gpu(d_input, d_input_second, srcSize, maxSize, noOfImages, handle);
+        break;
+    case 28:
+        test_case_name = "Accumulate Squared";
+        rppi_accumulate_squared_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, noOfImages, handle);
+        break;
+    case 29:
+        test_case_name = "Accumulate Weighted";
```

Comment:
Codacy found an issue: [Variable 'test_case_name' is assigned a value that is never used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)